internal/services/interface.go

```diff
@@ -122,6 +123,10 @@ type ModelDomainServices interface {
 	UnitState() *unitstateservice.Service
 	// Port returns the service for managing opened port ranges for units.
 	Port() *portservice.WatchableService
+	// Stub returns the stub service. A special service that collects temporary
+	// methods required for wiring together domains which are not completely
+	// implemented or wired up.
```

Comment:
Add a deprecated notice on this.

Reply:
![image](https://github.com/user-attachments/assets/2a380d0c-d050-48a9-b4ca-c3a127fbfdd4)
Nice